src/chi/components/steps/steps.scss

```diff
@@ -59,33 +62,51 @@ $breakpoints: ('sm', 'md', 'lg', 'xl');
     }
 
     & > a {
-      color: $brand-color;
-      font-weight: 600;
+      font-weight: $font-weight-semibold;
       margin-left: 0.5rem;
```

Comment:
![image](https://user-images.githubusercontent.com/130654/90158662-2070cf00-dd44-11ea-99ad-b93f80699c8e.png)
There's a bug with the margin-left here. It's not respecting the full 8px (off by 3px) due to the way the circle is structured. You can visualize better by setting the margin-left value to 0. Changing to 0.75rem (12px) appears to be the equivalent of 0.5rem (8px). We would also need to increase the margin between the arrow and the circle to ensure they're even.
![image](https://user-images.githubusercontent.com/130654/90158721-37afbc80-dd44-11ea-96f9-d16cd6293ecf.png)